analyzer/src/funTest/assets/projects/synthetic/glide-expected-output.yml

```diff
@@ -18,13 +18,13 @@ project:
   scopes:
   - name: "default"
     dependencies:
-    - id: "GoDep::github.com/mattn/go-isatty:v0.0.12"
+    - id: "Go::github.com/mattn/go-isatty:v0.0.12"
       linkage: "STATIC"
-    - id: "GoDep::golang.org/x/sys:05986578812163b26672dabd9b425240ae2bb0ad"
+    - id: "Go::golang.org/x/sys:05986578812163b26672dabd9b425240ae2bb0ad"
       linkage: "STATIC"
 packages:
-- id: "GoDep::github.com/mattn/go-isatty:v0.0.12"
-  purl: "pkg:golang/github.com%2Fmattn%2Fgo-isatty@v0.0.12"
+- id: "Go::github.com/mattn/go-isatty:v0.0.12"
+  purl: "pkg:go/github.com%2Fmattn%2Fgo-isatty@v0.0.12"
```

Comment:
The purl specification lists `golang` in their examples, I think we should comply to this specification and continue to use `golang` for PURLs.
https://github.com/package-url/purl-spec#some-purl-examples

Reply:
This is IMO out of scope of this commit, because it wasn't golang before and isn't golang after. 
I've discovered this issue with the types we used in PURL as well already. In today's dev meeting I promised to file a GitHub issue to address it (the problem is not just for Go packages). Agreed?